pkg/apis/agones/v1/gameserver.go

```diff
@@ -745,9 +747,10 @@ func (gs *GameServer) Pod(apiHooks APIHooks, sidecars ...corev1.Container) (*cor
 	}
 
 	gs.podObjectMeta(pod)
-	for _, p := range gs.Spec.Ports {
-		var hostPort int32
 
+	passthroughContainerPortMap := make(map[string][]string)
```

Comment:
why string and not int?

Reply:
True, it can be an int and that will save some conversions.